### ISSUE TITLE
Image renders locally but not in dev

### ISSUE THREAD

**Issue by cernst122** [Spelman-College/spelman-dashboard#201]
Our new Adobe stock images aren't rendering in the deployed version of the app and don't appear to be loading as a src:
![image](https://github.com/Spelman-College/spelman-dashboard/assets/13266735/6ee27047-a106-4dc0-a1e6-7c75e853861b)

They live in the same directory as /dots.svg and /stripes.svg and all are loaded as relative paths, so I'm not sure why this is happening.

**Comment by cernst122** [comment]
AWS logs:
```
2023-09-14T18:04:12.661Z [INFO]: # Executing command: npm run build
2023-09-14T18:04:12.992Z [INFO]: > spelman-dashboard@0.0.0 build
                                 > run-p type-check build-only
2023-09-14T18:04:13.441Z [INFO]: > spelman-dashboard@0.0.0 type-check
                                 > vue-tsc --noEmit
2023-09-14T18:04:13.442Z [INFO]: > spelman-dashboard@0.0.0 build-only
                                 > vite build
2023-09-14T18:04:14.183Z [INFO]: vite v4.4.9 building for production...
2023-09-14T18:04:14.551Z [INFO]: transforming...
2023-09-14T18:04:16.455Z [WARNING]: <path-to-image> referenced in /codebuild/output/src726366552/src/spelman-dashboard/frontend/src/views/reads/IndividualReadsView.vue?vue&type=style&index=0&scoped=7a8329ac&lang.css didn't resolve at build time, it will remain unchanged to be resolved at runtime
2023-09-14T18:04:16.541Z [WARNING]: <path-to-image> referenced in /codebuild/output/src726366552/src/spelman-dashboard/frontend/src/components/about/Vision.vue?vue&type=style&index=0&scoped=a8e3621f&lang.css didn't resolve at build time, it will remain unchanged to be resolved at runtime
2023-09-14T18:04:21.267Z [INFO]: ✓ 920 modules transformed.
2023-09-14T18:04:21.865Z [INFO]: rendering chunks...
2023-09-14T18:04:21.882Z [INFO]: computing gzip size...
2023-09-14T18:04:21.917Z [INFO]: dist/index.html                                                    0.84 kB │ gzip:   0.44 kB
2023-09-14T18:04:21.918Z [INFO]: dist/assets/roboto-v20-latin-ext_latin-regular-b86b128b.woff2     22.64 kB
                                 dist/assets/roboto-v20-latin-ext_latin-700-8d9364a0.woff2         22.72 kB
                                 dist/assets/roboto-v20-latin-ext_latin-500-fa074f87.woff2         22.73 kB
                                 dist/assets/roboto-v20-latin-ext_latin-regular-e70a908b.woff      29.04 kB
                                 dist/assets/roboto-v20-latin-ext_latin-500-d092ad8e.woff          29.08 kB
                                 dist/assets/roboto-v20-latin-ext_latin-700-e24c2752.woff          29.09 kB
                                 dist/assets/index-d16e5a80.css                                   580.84 kB │ gzip:  60.83 kB
                                 dist/assets/index-09bbef8f.js                                  1,284.41 kB │ gzip: 290.90 kB
2023-09-14T18:04:21.919Z [WARNING]: (!) Some chunks are larger than 500 kBs after minification. Consider:
                                    - Using dynamic import() to code-split the application
                                    - Use build.rollupOptions.output.manualChunks to improve chunking: https://rollupjs.org/configuration-options/#output-manualchunks
                                    - Adjust chunk size limit for this warning via build.chunkSizeWarningLimit.
2023-09-14T18:04:21.919Z [INFO]: ✓ built in 7.73s
2023-09-14T18:04:21.954Z [INFO]: # Completed phase: build
2023-09-14T18:04:21.956Z [INFO]: ## Build completed successfully
2023-09-14T18:04:21.957Z [INFO]: # Starting caching...
2023-09-14T18:04:21.960Z [INFO]: # Creating cache artifact...
2023-09-14T18:04:21.962Z [INFO]: # Created cache artifact
2023-09-14T18:04:21.962Z [INFO]: # Uploading cache artifact...
2023-09-14T18:04:22.130Z [INFO]: # Uploaded cache artifact
2023-09-14T18:04:22.130Z [INFO]: # Caching completed

![image](https://github.com/Spelman-College/spelman-dashboard/assets/13266735/a347b59c-2edf-407e-b8d0-46ea100c20e7)

```

**Comment by cernst122** [comment]
Fixed by #203 .
